src/commands.c

```diff
@@ -511,6 +513,22 @@ split_quoted (const gchar *src, const gboolean unquote)
     return argv;
 }
 
+static gchar *
+unescape (gchar *src) {
+    gchar *s = src, *p = src;
```

Comment:
Never did like this style of declarations. Two lines make the pointer thing clearer. Also, lone brace line for functions.